examples/chart/teleport/templates/deployment.yaml

```diff
@@ -38,7 +38,17 @@ spec:
 {{- if not .Values.proxy.tls.enabled }}
         - --insecure-no-tls
 {{- end }}
+        - --nodename=$(MY_NODE_NAME)
```

Comment:
this part could be problematic because a deployment can have multiple pods with teleport, which will cause a conflict with node names, is it required?

Reply:
@drewwells I agree with @klizhentas here - I think that in a case where there are multiple replicas (which is supported by this Helm chart, see https://github.com/gravitational/teleport/blob/master/examples/chart/teleport/templates/deployment.yaml#L11) then this has the potential to cause conflicts by giving multiple nodes the same name.